server/activate_hooks.go

```diff
@@ -54,7 +54,7 @@ func (p *Plugin) OnActivate() error {
 				"OnActivate: %s", manifest.Id,
 			),
 			Type: "custom_demo_plugin",
-			Props: map[string]interface{}{
+			Props: model.StringInterface{
```

Comment:
3/5, but I think we should stick with `map[string]interface{}` here instead of `model.StringInterface`. I think it's addition to `model` was premature: we don't add any methods to the type, and even methods like `StringInterfaceToJson` take a `map[string]interface{}`.

I suspect it will be less confusing for plugin authors to use the idiomatic type definition than an alias we use inconsistently.

Reply:
I get your point. Thanks for the review! I reverted my changes.